publishable/database/migrations/2016_01_01_000000_add_user_avatar_and_role_id.php

```diff
@@ -6,25 +6,23 @@ class AddUserAvatar extends Migration
 {
     /**
```

Comment:
Btw, I do not think ignoring the drop part is such a good idea? People might use it to uninstall Voyager.